virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -28,6 +28,8 @@ In {VirtProductName} clusters installed using installer-provisioned infrastructu
 
 * To use proxy with {VirtProductName}, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager].
 
+* When importing a virtual machines to a cluster or live migrating a virtual machine within a cluster, label nodes based on the CPU vendor so that all virtual machines in a pod use the same type of CPU. For example, virtual machines using AMD processors should not be placed on the same node as virtual machines that use Intel processors. xref:../../nodes/scheduling/nodes-scheduler-node-affinity.adoc#nodes-scheduler-node-affinity-configuring-required_nodes-scheduler-node-affinity[Configure a required node affinity rule] using the `Vendor` label to ensure virtual machines with different CPUs are not mixed on the same node.
```

Comment:
There's no "when live migrating". The person defining the VM (be it via import, wizard or yaml) does not control or even know when their VM is going to migrate. Also, there is no "all virtual machines in a pod". The text should
 * state the problem: two vendors in a cluster is a mess
 * request to label each node based on its vendor
 * request to label each VM, effectively slicing the cluster in two

suggestion:
If your cluster consists more than a single CPU vendor, VMs may attempt to live-migrate from one to another and likely to fail. To avoid this, you should label your nodes with a `Vendor=Intel` or `Vendor=AMD` label (add link, please) and set node affinity on your VMs so that each can run only on one vendor.

Reply:
OK, thanks @dankenigsberg. By way of explanation (not excuse) I got the first sentence from your prior comment _You can either import a VM to cluster, or live-migrate a VM in a cluster. I suppose this release note refers to the latter, but typical users do not choose to migrate, they can do nothing "before" migrations._ I also made the assumption that since "before" was not correct as I'd suspected, "when" was appropriate. My bad. Revising as you have suggested. Here is my wording based on what you just gave me. If you approve, please remove the hold label. Thanks again.

_* If your cluster uses virtual machines from multiple CPU vendors, live migration failures can occur. For example, a virtual machine with an AMD CPU may attempt to live-migrate to a virtual machine with an Intel CPU and likely fail migration. To avoid this, label nodes with a vendor-specific label, such as `Vendor=Intel` or `Vendor=AMD`, and set node affinity on your virtual machines to ensure successful migration. See xref:../../nodes/scheduling/nodes-scheduler-node-affinity.adoc#nodes-scheduler-node-affinity-configuring-required_nodes-scheduler-node-affinity[Configuring a required node affinity rule] for more information._

It sounds like we have supported the `Vendor=` label in OCP for some time. If so, I will target this for 2.4, 2.5, 2.6 and 4.8 if that makes sense (I will need to check to see if we are still updating 2.4). 

cc: @ILpinto @ctomasko @nunzy1